src/cmd-run

```diff
@@ -184,13 +172,38 @@ EOF
         exit 1
     fi
 fi
-set -- -fw_cfg name=opt/com.coreos/config,file="${IGNITION_CONFIG_FILE}" "$@"
+
+if [ -z "${VM_PERSIST_IMG}" ]; then
+    echo "Creating temporary image"
+    VM_IMG=$(mktemp -p "${TMPDIR:-/var/tmp}")
+    qemu-img create -q -f qcow2 -b "${VM_DISK}" "${VM_IMG}"
+else
+    echo "Re-using existing ${VM_PERSIST_IMG}"
+    VM_IMG=${VM_PERSIST_IMG}
+fi
```

Comment:
Doesn't this new logic break `--persist-img` though? Since we're no longer doing `qemu-img create` on the first run anymore, right?

Reply:
yeah I guess so - TBH this logic kinda makes my head hurt. Would be nice to simplify it if possible or add comments where necessary.

Reply:
Oh... so it is suposed to be managed by this code, I miss understood that part of code. Variable naming has been bit confusing. What do you want me to do about it? Another PR?

Reply:
@dustymabe  reasons for this block is that gustfish doesn't like deleted files as backing storage. Other has been to rename the persistent image to some thing more descriptive, because the "persistent" one has been in most case just temporal and caused some confusion for me. It is bit unfortunate this got caught post review.

Reply:
@jcajka - yep this logic was needed. It just required me to read it a few times. Some comments would help I think.